src/components/Reusables/CompactDeletableListItem/CompactDeletableListItem.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  Button,
+  ButtonVariant,
+  ListItem,
+  ButtonColor,
+  SvgIcon,
+} from '@altinn/altinn-design-system';
+import { useTranslation } from 'react-i18next';
+
+import { ReactComponent as MinusCircle } from '@/assets/MinusCircle.svg';
+
+import classes from './CompactDeletableListItem.module.css';
+
+export interface DeletableListItemProps {
+  removeCallback?: (() => void) | null;
+  firstText: string;
+  secondText: string;
+  startIcon?: React.ReactNode;
+}
+
+export const CompactDeletableListItem = ({
+  removeCallback,
+  firstText,
+  secondText,
+  startIcon,
```

Comment:
Wouln't Icon be sufficient?

Reply:
I think the prop should say something about where it the icon is placed

Reply:
Does it need to when there is only one icon?
The Page-component doesn't specify this (and I wouldn't expect it to either)

Reply:
Maybe not, I'll change it to icon